veg2hab/vegkartering.py

```diff
@@ -1195,7 +1195,8 @@ def _multi_col_to_vegtype(row: pd.Series) -> List[VegTypeInfo]:
             vvn = row.get(f"EDIT_VvN{idx}", "")
             vvn = "" if pd.isnull(vvn) else str(vvn)
             perc = row.get(f"EDIT_perc{idx}", None)
-            if sbb == "" and vvn == "":
+            # if all are empty, we're done for this row
+            if sbb == "" and vvn == "" and pd.isna(perc):
```

Comment:
is dit wel eens gebeurd? dat perc niet leeg is en sbb en vvn wel?

Reply:
Als er voor een rVvN geen sbb/vvn wordt gevonden komt het voor, dat is ook wanneer jakko het tegenkwam.